src/PassportServiceProvider.php

```diff
@@ -310,7 +310,8 @@ protected function registerResourceServer()
         $this->app->singleton(ResourceServer::class, function ($container) {
             return new ResourceServer(
                 $container->make(Bridge\AccessTokenRepository::class),
-                $this->makeCryptKey('public')
+                $this->makeCryptKey('public'),
```

Comment:
@driesvints could this be improved like this or am I overthinking it? [ResourceServer.php#L62-L64](https://github.com/thephpleague/oauth2-server/blob/43cd4d406906c6be5c8de2cee9bd3ad3753544ef/src/ResourceServer.php#L62-L64)
```php
$this->app->singleton(ResourceServer::class, function ($container) {
    $AccessTokenRepository = $container->make(Bridge\AccessTokenRepository::class);

    return new ResourceServer(
        $AccessTokenRepository,
        $this->makeCryptKey('public'),
        new Passport::$bearerTokenValidator($AccessTokenRepository)
    );
});
```

Reply:
Yeah, I think that's a good idea 👍

Reply:
@driesvints done!